var/spack/repos/builtin/packages/httpress/package.py

```diff
@@ -0,0 +1,28 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Httpress(MakefilePackage):
+    """High performance HTTP server stress & benchmark utility"""
+
+    homepage = "https://bitbucket.org/yarosla/httpress/wiki/Home"
+    url      = "https://bitbucket.org/yarosla/httpress/get/f27fa1949044.zip"
+
+    def url_for_version(self, version):
+        url = "https://bitbucket.org/yarosla/httpress/get/f27fa1949044.zip"
+        return url
```

Comment:
Why don't we add v1.1.0? It seems they are referring to the same code as far as I can understand from [here](https://bitbucket.org/yarosla/httpress/commits/)

Reply:
It looks like no one maintain this repo now.
And we must use `hg clone xxx ` instead of `git clone xxx` to get the repo and checkout v1.1.0.
So I'm not sure how to get v1.1.0 version by spack framework.
Any suggestion to handle this situation?

Reply:
Spack has full support for fetching from Mercurial repos: https://spack.readthedocs.io/en/latest/packaging_guide.html#mercurial

Reply:
fix it in https://github.com/spack/spack/pull/15935/commits/5c52e3c099c87b22f74c540e828ef850e5e6b560